actpool/actpool.go

```diff
@@ -40,6 +40,8 @@ var (
 		Name: "iotex_actpool_rejection_metrics",
 		Help: "actpool metrics.",
 	}, []string{"type"})
+	// ErrGasTooHigh error when the intrinsic gas of an action is too high
+	ErrGasTooHigh = errors.New("action gas is too high")
```

Comment:
move to `action/const.go`? that's where all other errors like `ErrTxPoolOverflow, ErrNonceTooHigh` are defined